recipes/out_ptr/all/conanfile.py

```diff
@@ -1,31 +1,32 @@
+from conan import ConanFile
+from conan.tools.files import copy, get
+from conan.tools.layout import basic_layout
 import os
 
-from conan import ConanFile
-from conan.tools.files import get
+required_conan_version = ">=1.46.0"
 
-required_conan_version = ">=1.45.0"
 
 class OutPtrConan(ConanFile):
     name = "out_ptr"
-
-    # Optional metadata
     license = "Apache-2.0"
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/soasis/out_ptr"
     description = "a C++11 implementation of std::out_ptr (p1132), as a standalone library"
     topics = ("utility", "backport")
     settings = "os", "arch", "build_type", "compiler"
-
-    @property
-    def _source_subfolder(self):
-        return "source_subfolder"
+    no_copy_source = True
 
     def package_id(self):
         self.info.header_only()
```

Comment:
```suggestion
        self.info.clear()
```
Please use `clear()`.

Reply:
yep, but I also need to bump required_conan_version to 1.50.0